kubernetes/reporter/src/main/java/org/arquillian/cube/kubernetes/reporter/TakeKubernetesResourcesInformation.java

```diff
@@ -130,20 +139,53 @@ private TableEntry createTableEntryWithTHead(String name, String... heads) {
         return tableEntry;
     }
 
-    private FileEntry getFileForResourcesConfiguration(Configuration configuration, ReporterConfiguration reporterConfiguration) throws IOException {
-        String path = configuration.getEnvironmentConfigUrl().getPath();
+    private ArrayList<FileEntry> getFilesForResourcesConfiguration(Session session, Configuration configuration, ReporterConfiguration reporterConfiguration) throws IOException {
+        ArrayList<FileEntry> fileEntries = new ArrayList<>();
+        URL environmentConfigUrl = configuration.getEnvironmentConfigUrl();
+
+        if (environmentConfigUrl != null) {
+            fileEntries.add(getFileForResourcesConfiguration(environmentConfigUrl, reporterConfiguration));
+        }
+
+        if (configuration.isEnvironmentInitEnabled()) {
+            List<URL> dependencyUrls = !configuration.getEnvironmentDependencies().isEmpty() ? configuration.getEnvironmentDependencies() : dependencyResolver.get().resolve(session);
+
+            for (URL dependencyUrl : dependencyUrls) {
+                fileEntries.add(getFileForResourcesConfiguration(dependencyUrl, reporterConfiguration));
+            }
+        }
+
+        return fileEntries;
+    }
+
+    private FileEntry getFileForResourcesConfiguration(URL url, ReporterConfiguration reporterConfiguration) throws IOException {
+
 
         final Path rootDir = Paths.get(reporterConfiguration.getRootDir().getAbsolutePath());
-        final Path relativize = rootDir.relativize(Paths.get(path));
+        final String filePath = relativizePath(url, rootDir);
 
         FileEntry fileEntry = new FileEntry();
-        fileEntry.setPath(relativize.toString());
+        fileEntry.setPath(filePath);
         fileEntry.setMessage(CONFIGURATION);
-        fileEntry.setType(getFileType(relativize));
+        fileEntry.setType(getFileType(filePath));
 
         return fileEntry;
     }
 
+    private String relativizePath(URL url, Path rootDir) {
```

Comment:
Yeah to convert uri to path as it depends in filesystem, it will remove one backslash on linux
so `http://foo.com` will convert to `http:/foo.com`. So to avoid this. continuing with the same.